src/localization/fr-FR.json

```diff
@@ -2,8 +2,12 @@
   "hello": "H-e-l-l-o-!",
   "about-us": "A-b-o-u-t- -U-s",
   "about-us-content": "L-o-r-e-m- -i-p-s-u-m- -d-o-l-o-r- -s-i-t- -a-m-e-t-,- -c-o-n-s-e-c-t-e-t-u-r- -a-d-i-p-i-s-c-i-n-g- -e-l-i-t-,- -s-e-d- -d-o- -e-i-u-s-m-o-d- -t-e-m-p-o-r- -i-n-c-i-d-i-d-u-n-t- -u-t- -l-a-b-o-r-e- -e-t- -d-o-l-o-r-e- -m-a-g-n-a- -a-l-i-q-u-a-.- -U-t- -e-n-i-m- -a-d- -m-i-n-i-m- -v-e-n-i-a-m-,- -q-u-i-s- -n-o-s-t-r-u-d- -e-x-e-r-c-i-t-a-t-i-o-n- -u-l-l-a-m-c-o- -l-a-b-o-r-i-s- -n-i-s-i- -u-t- -a-l-i-q-u-i-p- -e-x- -e-a- -c-o-m-m-o-d-o- -c-o-n-s-e-q-u-a-t-.- -D-u-i-s- -a-u-t-e- -i-r-u-r-e- -d-o-l-o-r- -i-n- -r-e-p-r-e-h-e-n-d-e-r-i-t- -i-n- -v-o-l-u-p-t-a-t-e- -v-e-l-i-t- -e-s-s-e- -c-i-l-l-u-m- -d-o-l-o-r-e- -e-u- -f-u-g-i-a-t- -n-u-l-l-a- -p-a-r-i-a-t-u-r-.- -E-x-c-e-p-t-e-u-r- -s-i-n-t- -o-c-c-a-e-c-a-t- -c-u-p-i-d-a-t-a-t- -n-o-n- -p-r-o-i-d-e-n-t-,- -s-u-n-t- -i-n- -c-u-l-p-a- -q-u-i- -o-f-f-i-c-i-a- -d-e-s-e-r-u-n-t- -m-o-l-l-i-t- -a-n-i-m- -i-d- -e-s-t- -l-a-b-o-r-u-m-.",
+  "add-promotion": "Add Promotion",
   "discount-at-checkout": "D-i-s-c-o-u-n-t- -a-t- -C-h-e-c-k-o-u-t",
   "applied-promotions": "A-p-p-l-i-e-d- -P-r-o-m-o-t-i-o-n-s",
+  "apply-promotion": "A-p-p-l-y P-r-o-m-o-t-i-o-n",
+  "promotion-invalid": "P-l-e-a-s-e- -e-n-t-e-r- -v-a-l-i-d- -p-r-o-m-o-t-i-o-n-",
+  "add-promotion-title": "-E-n-t-e-r- -C-o-u-p-o-n- -C-o-d-e-",
```

Comment:
Please don't put leading and trailing hyphens for these. The file can be generated, but looks like these were added manually 🙂

Reply:
I'll fix this too for you then merge